docs/fio.md

```diff
@@ -0,0 +1,299 @@
+# FIO Benchmark
+
+[FIO](https://github.com/axboe/fio) or Flexible IO Tester is a tool that would be able to simulate a given I/O workload without resorting to writing a tailored test case again and again.
+
+FIO spawns a number of threads or processes doing a particular type of I/O action as specified by the user. FIO takes a number of global parameters, each inherited by the thread unless otherwise parameters given to them overriding that setting is given. The typical use of FIO is to write a job file matching the I/O load one wants to simulate.
+
+## Running FIO Benchmark
+
+Once the operator has been installed following the instructions, one needs to modify the [cr.yaml](../resources/crds/benchmark_v1alpha1_benchmark_cr.yaml) to run either sequential, random or custom workload. For custom workload one can provide URL of http server where FIO job file is present.
+
+The FIO section in [cr.yaml](../resources/crds/benchmark_v1alpha1_benchmark_cr.yaml) would look like this.
+
+```yaml
+apiVersion: benchmark.example.com/v1alpha1
+kind: Benchmark
+metadata:
+  name: example-benchmark
+spec:
+  fio:
+    # To disable fio, set clients to 0
+    job: seq # either of  seq, rand or custom
+    clients: 2
+    jobname: seq # either of seq, rand or custom
+    get_url: http://perf1.perf.lab.eng.bos.redhat.com/pub/shberry/fio/custom.fio #If Job is custom, provide URL of job here
+    bs: 64k # provide only if job is seq or rand
+    iodepth: 4 # provide only if job is seq or rand
+    runtime: 60 # provide only if job is seq or rand
+    numjobs: 2 # provide only if job is seq or rand
+    filesize: 2 # File size in GB,  provide only if job is seq or rand
+    storageclass: rook-ceph-block # Provide if PV is needed
```

Comment:
Also it'd be helpful to add some documentation for the user to understand what adding the pvs would do wrt to job.

Reply:
Resolved as it's mentioned in the note below